pkg/api/v1/cluster_types.go

```diff
@@ -236,7 +236,7 @@ type RackSpec struct {
 	// Resources the Scylla container will use.
 	Resources corev1.ResourceRequirements `json:"resources"`
 	// AgentResources which Agent container will use.
-	// +kubebuilder:default:={requests: {cpu: "50m", memory: "10M"}, limits: {cpu: "1", memory: "200M"}}
+	// +kubebuilder:default:={requests: {cpu: "50m", memory: "10M"}}
```

Comment:
I am all in for requests only, but doesn't our current validation prevent that?

Reply:
I double checked and AgentResources aren't validated

Reply:
oh, agent